includes/config.php

```diff
@@ -41,6 +41,40 @@ function research_define_customizer_fields( $wp_customize ) {
 			'section'     => RESEARCH_THEME_CUSTOMIZER_PREFIX . 'clusters'
 		)
 	);
+
+	$wp_customize->add_setting(
+		'cluster_fallback_bg'
+	);
+
+	$wp_customize->add_control(
+		new WP_Customize_Image_Control(
+			$wp_customize,
+			'cluster_fallback_bg',
+			array(
+				'label'    => __( 'Upload a fallback header image for clusters' ),
+				'section'  => RESEARCH_THEME_CUSTOMIZER_PREFIX . 'clusters',
+				'height'   => 1600,
```

Comment:
Height and width values appear to be inverted.  Also, the height for header images inherited from the UCF WP Theme is 500, not 550--not sure if this is an intentional change.

Reply:
Updated the constraints.. I'll have to update the header images I've source so far for this.